linux_os/guide/services/ntp/service_timesyncd_enabled/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+title: 'Enable systemd_timesyncd'
+
+description: '{{{ describe_service_enable(service="systemd_timesyncd") }}}'
+
+rationale: |-
+    Enabling the <tt>systemd_timesyncd</tt> service ensures that this host 
+    uses the ntp protocol to fetch time data from a ntp server.
+    Synchronizing time is essential for authentication
+    services such as Kerberos, but it is also important for maintaining accurate
+    logs and auditing possible security breaches.
+    <br /><br />
+    deprecated. Additional information on Ubuntu network time protocol is 
```

Comment:
This doesn't look right. Do you mean to say that ntp/ntpdate services are deprecated?

Reply:
no, the deprecation of ntpdate is talked about in the ntp_enabled test.
The link pointed to in rule.yml shows that systemd_timesyncd is the new standard for ntp clients on ubuntu1604. The standard ntp server is also not ntpd anymore but no need to mention that in this rule.yml

Reply:
@AxelNennker then I think you forgot to remove `deprecated.`